ui/pages/confirmations/components/confirm/title/title.tsx

```diff
@@ -80,14 +81,19 @@ const getTitle = (
   pending?: boolean,
   primaryType?: keyof typeof TypedSignSignaturePrimaryTypes,
   tokenStandard?: string,
+  isUpgrade?: boolean,
 ) => {
   if (pending) {
     return '';
   }
 
   switch (confirmation?.type) {
     case TransactionType.contractInteraction:
+      return t('confirmTitleTransaction');
     case TransactionType.batch:
+      if (isUpgrade) {
+        return t('confirmTitleDelegationRevoke');
```

Comment:
Is this a typo? Upgrade uses the revoke label?

Reply:
May be I need to rename the label, actually both upgrade and revoke have same title in new designs.

Reply:
Fixed in the PR